### PR TITLE
Added Go-Back and Scroll-To-top functionality to Course Description page

### PR DESCRIPTION
**Pull request by Abhijay007**
**Describe the changes you've made**

Added Go-back on wishlist page functionality on click of back arrow button in course description page and Added scroll to top button also

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

`NA`

**Additional context (OPTIONAL)**

***After***

after-courseDesc

**Checklist**

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by faisaljamil25** [comment]
Hi @Abhijay007 
I have changed the `CourseDetail.js` location in PR #177 so it would raise conflicts if either one of our PR's gets merged. So to resolve this issue you can move the file inside `CourseDetail` folder and rename it to `index.js`

File: src/layouts/MainLayout/BottomNav.css [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
 }
 
 @media screen and (max-width: 321px) and (min-width: 320px) {
   .Mui-selected {
-    font-size: 0.67rem !important;
+    font-size: 0.6rem !important;
   }
   .MuiBottomNavigationAction-label {
-    font-size: 0.66rem !important;
+    font-size: 0.6rem !important;
```
Comment (Abhishek-kumar09) [inline src/layouts/MainLayout/BottomNav.css:20]: revert this, fixed it in  the latest code

**Comment by Abhijay007** [comment]
Ok

**Comment by Abhijay007** [comment]
> Hi @Abhijay007
> I have changed the `CourseDetail.js` location in PR #177 so it would raise conflicts if either one of our PR's gets merged. So to resolve this issue you can move the file inside `CourseDetail` folder and rename it to `index.js`

ok, @faisaljamil25 I will do that thanks for letting me know about it !!

**Pull request merged**